.github/workflows/build.yml

```diff
@@ -23,7 +23,7 @@ env:
   mysqlLegacySqlMode: ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION
   # Currently no ONLY_FULL_GROUP_BY, see #1167:
   maxConnections: 255
-  "ConnectionStrings__Default": server=127.0.0.1;user id=root;password=Password12!;port=3306;sslmode=Preferred;database=Orleans;
+  "ConnectionStrings__Default": server=127.0.0.1;user id=root;password=Password12!;port=3306;sslmode=Preferred;database=Orleans;DefaultCommandTimeout=10000000;
```

Comment:
curious why this change is required?

Reply:
Good call out -- that connection string only is for running the tests of the migrations.

Right now it doesn't matter as there is no data in the DB during the migration. Once functional tests are added, we'll have seeded data in the DB and the migrations will operate on that DB and larger data sets take more time. 

That said, I'll remove it for now and leave it for when it actually becomes a problem.